config-templates/redhat/odf-local/changelog.json

```diff
@@ -1,23 +1,41 @@
 [
+    {
+        "4.16": {
+            "status": "supported",
+            "latest-revision": {
+                "revision": "1",
+                "release-date": "12 October 2024",
+                "cves": "",
+                "golang": "1.22.7",
+                "ubi": "",
+                "new-features": [
+                    "ODF 4.16 initial release"
+                ],
+                "fixes": [
+                ]
+            }
+        }
+    },
     {
         "4.15": {
             "status": "supported",
             "latest-revision": {
-                "revision": "5",
-                "release-date": "04 September 2024",
-                "cves": "CVE-2024-2398, CVE-2024-37370, CVE-2024-37371",
+                "revision": "6",
+                "release-date": "12 October 2024",
+                "cves": "",
                 "golang": "1.21.13",
                 "ubi": "",
                 "new-features": [
                 ],
                 "fixes": [
+                    "Sidecar golang updated to gv1.22.7-community-20240916"
```

Comment:
Can you please clarify if sidecar images are updated to Go 1.22.7 but ODF image is still based on Go 1.21.13?

Reply:
yes, ODF images are still on 1.21.13